library/std/src/sys/windows/args.rs

```diff
@@ -21,6 +21,14 @@ use crate::vec;
 
 use core::iter;
 
+/// This is the const equivalent to `NonZeroU16::new(n).unwrap()`
+const fn non_zero_u16(n: u16) -> NonZeroU16 {
```

Comment:
Maybe left FIXME with comment that this should be replaced back when feature will be stabilized?

Reply:
Done!